includes/migrations/class-delete-active-webhook.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Class Delete_Active_Webhook
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Migrations;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class Delete_Active_Webhook
+ *
+ * Fires an event on plugin upgrade to delete the avtive webhook.
+ * Runs only once. We want to know whether existing install had it
+ * enabled before the current version.
+ */
+class Delete_Active_Webhook {
```

Comment:
Not a blocker, but wondering if this class should include mention of WooPay/platform checkout in its name (e.g. `Delete_Active_Platform_Checkout_Webhook`) just for clarity.

Reply:
Good call on the sanitized naming!